webservices/args.py

```diff
@@ -377,6 +377,7 @@ def make_seek_args(field=fields.Int, description=None):
     'min_total_contributions': Currency(description=docs.MIN_FILTER),
     'max_total_contributions': Currency(description=docs.MAX_FILTER),
     'type': fields.List(fields.Str, description=docs.COMMITTEE_TYPE),
```

Comment:
`type` filter is replaced by `committee_type` filter. `type` filter is no longer needed for this endpoint. You can remove this line.

Reply:
Will be done after users are notified due to breaking change as per our deprecation policy and my convo with @pkfec